src/lib/stack/stack-constants.ts

```diff
@@ -0,0 +1,32 @@
+import { COMPONENT_NAME_PREFIX } from '../constants';
+
+const elementName = `${COMPONENT_NAME_PREFIX}stack`;
+
+const classes = {
+  DEFAULT: 'forge-stack'
+};
+
+const attributes = {
+  INLINE: 'inline',
+  WRAP: 'wrap',
+  STRETCH: 'stretch',
+  GAP: 'gap',
```

Comment:
Should this use the same terminology as CSS or would it be more appropriate to call this attribute `spacing` or something like that?

Reply:
Yeah I'm on board with sticking with `inline`